### PR TITLE
use dotenv file properly to get URL.

### PR DESCRIPTION
**Pull request by TheunisK**

### CODE REVIEW

File: src/client/components/Login.jsx [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+
+console.log(loginURL);
+
 const emptyDetails = {
     username: "",
     password: ""
 }
 
-const loginURL = "http://localhost:4000/user/login"; //need to update this URL when we try to run the fetch.
-
```
Comment (Can-Tina) [inline src/client/components/Login.jsx:9]: Goodbye hard coding

File: src/client/components/Login.jsx [hunk 2]
```diff
@@ -1,12 +1,16 @@
 import { useState } from "react";
 
+const URL = process.env.REACT_APP_API_URL;
```
Comment (Can-Tina) [inline src/client/components/Login.jsx:3]: Glad you got this sorted :)

File: src/client/components/Registration.jsx [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
 const emptyUser = {
     email: "",
     username: "",
     password: ""
 }
 
-const registerURL = "http://localhost:4000/user/register"; // need to update this URL for when we test the fetch.
-
 const Registration = () => {
     const [userDetails, setUserDetails] = useState(emptyUser);
```
Comment (Can-Tina) [inline src/client/components/Registration.jsx:16]: and the same here

**Comment by nathanSamosa** [comment]
all looks good to me :)

File: src/client/components/Login.jsx (hunk 1 above)
Comment (TheunisK) [reply src/client/components/Login.jsx:9]: Getting the hard code away was hard... Ironic!

Comment (glowkeeper) [reply src/client/components/Login.jsx:9]: nice work

**Comment by glowkeeper** [comment]
Approved

**Pull request merged**